diff --git a/build/commands/autolinkingOptions.js b/build/commands/autolinkingOptions.js
index ecd1f2c18a7b07b1d0a7c71a125e5f9f8b75a5ba..6d0105adc0b94d024d03922637b06a903cbb1a45 100644
--- a/build/commands/autolinkingOptions.js
+++ b/build/commands/autolinkingOptions.js
@@ -110,11 +110,17 @@ const parseExtraArgumentsOptions = (args) => {
 };
 const findPackageJsonPathAsync = async (commandRoot) => {
     const root = commandRoot || process.cwd();
-    for (let dir = root; path_1.default.dirname(dir) !== dir; dir = path_1.default.dirname(dir)) {
+    let dir = root;
+    while (true) {
         const file = path_1.default.resolve(dir, 'package.json');
         if (fs_1.default.existsSync(file)) {
             return file;
         }
+        const parent = path_1.default.dirname(dir);
+        if (parent === dir) {
+            break;
+        }
+        dir = parent;
     }
     throw new Error(`Couldn't find "package.json" up from path "${root}"`);
 };
